components/sysutils/sg3_utils/Makefile

```diff
@@ -48,21 +47,23 @@ CFLAGS += -std=c99 $(CPP_XPG6MODE)
 CONFIGURE_OPTIONS +=	--disable-static
 CONFIGURE_OPTIONS += CPPFLAGS="$(CPPFLAGS)"
 CONFIGURE_OPTIONS += CFLAGS="$(CFLAGS)"
+CONFIGURE_OPTIONS += --bindir=$(USRBINDIR)
```

Comment:
This is confusing. We get `--bindir=/usr/bin/amd64 --bindir=/usr/bin` in configure args. 
1) Libs going to /usr/lib/amd64 is fine (and correct for 64-bit libraries). 
2) Use PREFERRED_BITS=64 like in https://github.com/OpenIndiana/oi-userland/blob/b538287cfdbce379cd40a703c72a76f4983d15fa/components/sysutils/findutils/Makefile#L25 to switch CONFIGURE_BINDIR.